README.md

```diff
@@ -12,60 +12,65 @@
 
 Reaction’s out-of-the-box core features include:
 
--   Drag-and-drop merchandising
+-	One step cart and checkout
 -   Order processing
--   Payments
+-   Payments with Stripe
 -   Shipping
 -   Taxes
+-   Multi-currency support
 -   Discounts
--   Analytics
 -   Integration with dozens of third-party apps
--   See full list of features on our [Roadmap](https://reactioncommerce.com/roadmap)
+-   See full list of features on our [roadmap](https://reactioncommerce.com/roadmap)
 
 Since anything in our codebase can be extended, overwritten, or installed as a package, you may also develop, scale, and customize anything on our platform.
 
 # Getting started
 
 ### Requirements
 
-Reaction requires Meteor, Git, MongoDB, OS-specific build tools and optionally, ImageMagick. For step-by-step instructions, check out this [page](https://docs.reactioncommerce.com/reaction-docs/master/installation).
+Reaction requires Meteor, Git, Mongo DB, OS-specific build tools. For step-by-step instructions, follow the documentation for [OS X](https://docs.reactioncommerce.com/docs/next/installation-osx), [Windows](https://docs.reactioncommerce.com/docs/next/installation-windows) or [Linux](https://docs.reactioncommerce.com/docs/next/installation-linux).
 
-### Install and create your first store
+### Install and run with CLI
 
 Install the [Reaction CLI](https://github.com/reactioncommerce/reaction-cli) to get started with Reaction:
 
-```bash
+```sh
 npm install -g reaction-cli
 ```
 
 Create your store:
 
-```bash
+```sh
 reaction init
 cd reaction
 reaction
 ```
 
+Open `localhost:3000`
+
+Learn more on how to [configure your project](https://docs.reactioncommerce.com/reaction-docs/master/configuration). 
+
+Having installation issues? Check out our [troubleshooting docs](https://docs.reactioncommerce.com/docs/next/troubleshooting-development).
+
+### Install and run with Docker
+
 You can also run the app locally using [`docker-compose`](https://docs.docker.com/compose/) by running:
 
 ```sh
+docker network create reaction-api
```

Comment:
`reaction-api` is correct for `master`, but if you're merging this into 1.17 or 2.0 branches, it's now `docker network create api.reaction.localhost`